modular_skyrat/modules/mapping/_maps/RandomRuins/SpaceRuins/blackmarket.dmm

```diff
@@ -350,6 +361,7 @@
 /obj/machinery/atmospherics/pipe/smart/manifold4w/supply/hidden,
 /obj/effect/decal/cleanable/dirt/dust,
 /obj/item/crowbar,
+/obj/item/storage/backpack/duffelbag/syndie,
```

Comment:
Why a syndicate duffelbag specifically?

Reply:
Thematically appropriate and gives them more storage than a normal backpack.

Reply:
Not really appropriate given the black market dealer isn't with the syndicate.
At the same time there isn't a reason not to give them one either.

Reply:
It's basically a duffel with no slowdown, that's literally it tbh. Plus the label is just "suspicious duffelbag" in game anyway

Reply:
traitors / anyone with the syndicate faction can double examine to see it's actually a syndicate bag. Not thematically appropriate considering they're an entirely different group, plus I - being the person who implemented this under gandalf's direction - generally disapprove of BMTs operating solely out of their shuttle anyways.

Reply:
Fair but the fact the shuttle doesn't have any way to contain air is kinda... iffy

Reply:
It would be nice if the shuttle had those little built in holofans that bluespace shelters have, if that would be possible to do.

Reply:
Good point, I can do that instead later